docs/README.md

```diff
@@ -0,0 +1,624 @@
+
+### Obtain an ID Token
+
+In order to obtain an ID Token, use the `id_token` url from the [Discovery document](#api-discovery), which allows us to exchange our Refresh Token for a fresh ID Token.
+
+<a class="id_token-url"></a>
+
+Make a POST request with the following payload, injecting the Refresh Token as specified: `{"grant_type": "refresh_token", "refresh_token": <API Key>}`. Extract the `id_token` field from the response, which contains the ID Token, which can then be sent to the API.
+
+For example, with `curl` to make the request and `jq` to extract the field:
+
+```sh
+ID_TOKEN_URL="<ID Token Discovery URL>"
+API_KEY="<Refresh Token>"
+curl -fsSl -XPOST \
+     -H "Content-Type: application/json" \
+     -d "{\"grant_type\": \"refresh_token\", \"refresh_token\": \"$API_KEY\"}" \
+     "$ID_TOKEN_URL" \
+     | jq -r '.id_token'
+```
+
+You now have an ID Token that can be used with the AIM API!
+
+### Abuse and Privacy
+
+In order to prevent abuse and data leaks, Refresh Tokens must be stored securely. In particular, avoid unneeded sharing of Refresh Tokens or storing them in source files/source control.
+
+If you suspect your Refresh Token has been compromised, contact <mailto:AIM@thestrawgroup.com> as soon as possible. If abuse is suspected, TSG may disable Refresh Tokens immediately and follow up with the API contact.
+
+## Query API
+
+The Query API is the primary tool provided by the AIM API. It provides powerful data analysis across multiple dimensions of the AIM dataset.
+
+The Query API is comprised of four (4) primary components:
+- `aggregation`
+- `attribute`
+- `metric`
+- `normalization`
+
+Each component has a discovery endpoint to obtain the available items with full metadata. All query urls are based from the `warehouse` url in the [discovery document](#api-discovery):
+
+<code class="warehouse-url"></code>
+
+### Quickstart
+
+After [acquiring an ID Token](#obtain-an-id-token), start with a few simple API calls. The calls will use `curl` for demonstration, but of course, any HTTP client will do. In these examples, `BASE_URL` is set to <code class="warehouse-url"></code>. Any query results are for demonstration purposes only and do not represent real values.
+
+```
+# Inspect all attributes. Note the rich metadata describing the data type, filter config and values, among other things. These attributes determine how the data can be filtered and grouped.
+curl -H "Authorization: Bearer $ID_TOKEN" \
+        "$BASE_URL/attribute/"
+
+# Inspect all metrics. Notice that metrics have "availability" metadata describing what attributes and normalizations they support.
+curl -H "Authorization: Bearer $ID_TOKEN" \
+        "$BASE_URL/metric/"
+
+# Let's run a query to pull out volume data:
+curl -H "Authorization: Bearer $ID_TOKEN" \
+        "$BASE_URL/query?metrics=volume&filter=date=2018-01"
+# [
+#   {
+#     "volume":12864.75939
+#   }
+# ]
```

Comment:
Al:
> - What does the comment volume 12864.75939 actually imply?
> - In all samples it utilizes 2018-01, is that the required value that needs to be used or can that change?

Reply:
This is a fake "Volume Per Merchant" for 2018-01. "Per Merchant" is the default normalization, so doesn't have to be included in the URL, though can be or be set differently.

I just used 2018-01 for ease.

I've added some more date examples and moved up an example showing a different normalization closer, so hopefully that clarifies a bit more.

Do you have concrete copy changes I should make?